azplugins/integrate.py

```diff
@@ -158,3 +158,246 @@ def set_params(self, H=None, V=None, boundary=None):
 
         bc = self._process_boundary(self.boundary)
         self.cpp_method.geometry = _mpcd.SlitGeometry(self.H,self.V,bc)
+
+
+class sinusoidal_channel(_bounce_back):
+    """ NVE integration with bounce-back rules in a anti-symmetric sinusoidal channel.
+
+    Args:
+        group (:py:mod:`hoomd.group`): Group of particles on which to apply this method.
+        A (float): channel cosine amplitude
+        h (float): channel half-width
+        p (int):   channel periodicity
+        boundary : 'slip' or 'no_slip' boundary condition at wall (default: 'no_slip')
+
+    This integration method applies to particles in *group* in the symmetric
+    cosine channel geometry.
+
+    The boundary condition at the wall can be specified with *boundary* to either
+    `'slip'` or `'no_slip'`. A no-slip condition is more common for solid boundaries.
+    If the no-slip condition is combined with a body-force on the particles, a
+    flow field with be generated with a velocity close to zero at the walls.
+
+    Note:
+        It may be necessary to add additional 'ghost' particles near the boundaries
+        in order to correctly enforce the boundary conditions and to reduce
+        density fluctuations near the wall.
+
+    HOOMD uses a periodic simulation box, but the geometry imposes inherent
+    non-periodic boundary conditions. You **must** ensure that the box is
+    sufficiently large to enclose the geometry (i.e., :math:`L_z > 2H`). An
```

Comment:
This comment looks outdated
```suggestion
    sufficiently large to enclose the geometry. An
```